schedule/yast/maintenance/yast-mru-install-minimal-with-addons.yaml

```diff
@@ -1,26 +1,25 @@
 ---
 name: yast-mru-install-minimal-with-addons
 vars:
-  PATTERNS: base,minimal
+  PATTERNS: base,enhanced_base
   YUI_REST_API: 1
 schedule:
   - installation/bootloader_start
   - installation/setup_libyui
   - '{{installsles}}'
   - installation/licensing/accept_license
   - installation/registration/register_via_scc
-  - installation/module_registration/register_extensions_and_modules
+  - '{{register}}'
   - installation/add_on_product/add_maintenance_repos
   - installation/addon_products_sle
   - installation/partitioning/accept_proposed_layout
   - installation/clock_and_timezone/accept_timezone_configuration
   - installation/authentication/use_same_password_for_root
   - installation/authentication/default_user_simple_pwd
-  - installation/select_patterns
+  - installation/select_only_visible_patterns_from_top
   - '{{validate}}'
   - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/installation_overview
```

Comment:
This change looks not directly related, would be good to mention it in the commit.

Reply:
Changed to use libyui test modules. 
'- installation/installation_overview' and '- installation/disable_grub_timeout' don't use libyui